client/js/clientLogic.js

```diff
@@ -0,0 +1,280 @@
+'use strict';
+/*
+ * clientLogic.js
+ * Copyright(c) 2016 Universidad de Las Palmas de Gran Canaria
+ * Authors:
+ *   Jaisiel Santana <jaisiel@gmail.com>
+ *   Alejandro Sánchez <alemagox@gmail.com>
+ *   Pablo Fernández <pablo.fernandez@ulpgc.es>
+ * MIT Licensed
+
+  This module specifies the client application behaviour.
+  It connects restaurantsAPI with drawModule.
+
+*/
+
+/*exported clientLogic */
+
+var connectionsAPI;
+var drawModule;
+var restaurantsAPI;
+
+var clientLogic = (function() {
+
+  function showAllRestaurants() {
+    restaurantsAPI.getAllRestaurants(
+      function(response) { //success
+        var restaurants =
+          restaurantsAPI.simplifyRestaurantsFormat(response);
+        drawModule.addRestaurantstoMap(restaurants);
+      },
+      function(response) { //error
+        alert('Could not retrieve restaurants');
+        if (response) {
+          console.log(response);
+        }
+      }
+    );
+  }
+
+  function showOrganizationRestaurants(organization) {
+    restaurantsAPI.getOrganizationRestaurants(
+      organization,
+      function(response) { //success
+        var restaurants =
+          restaurantsAPI.simplifyRestaurantsFormat(response);
+        drawModule.addRestaurantstoMap(restaurants);
+      },
+      function(response) { //error
+        alert('Could not retrieve restaurants');
+        if (response) {
+          console.log(response);
```

Comment:
console.error 

and you would need to log something more meaningful

console.error('Error while xxx : ' , response.status, response.<whatever>), something that allows to debug what is happening